
2022-10-23 19:53:02.585 - [Debug] [Connection] Attacker connected: 59.1.226.211 | Client Identification: SSH-2.0-Go
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
(node:434153) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_3',
  containerIp: '10.0.3.51',
  mitmPort: 1002,
  autoAccessFixed: 1
}
2022-10-23 19:53:32.060 - [Info] MITM Version: 2
2022-10-23 19:53:32.061 - [Info] Auto Access Enabled: true
2022-10-23 19:53:32.062 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-23 19:53:32.069 - [Info] SSH man-in-the-middle server for 10.0.3.51 listening on 10.0.3.1:1002
2022-10-23 19:53:33.055 - [Debug] [Connection] Attacker connected: 43.139.28.174 | Client Identification: SSH-2.0-Go
2022-10-23 19:53:34.147 - [Debug] [Connection] Attacker connected: 223.171.91.191 | Client Identification: SSH-2.0-Go
2022-10-23 19:53:34.450 - [Debug] [Auth] Attacker 43.139.28.174 trying to authenticate with 'none'
2022-10-23 19:53:34.450 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 19:53:34.939 - [Debug] [Auth] Attacker 43.139.28.174 trying to authenticate with 'password'
2022-10-23 19:53:34.940 - [Debug] [Auto Access] Attacker: 43.139.28.174, Threshold: 1, Attempts: 1
2022-10-23 19:53:34.941 - [Debug] [Auto Access] Compromising the honeypot
2022-10-23 19:53:34.941 - [Debug] [Auto Access] Adding the following credentials: 'root:1'
2022-10-23 19:53:35.006 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2022-10-23 19:53:35.008 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-23 19:53:35.008 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.51
2022-10-23 19:53:35.050 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2022-10-23 19:53:35.598 - [Debug] [Connection] Attacker connected: 222.117.95.174 | Client Identification: SSH-2.0-Go
2022-10-23 19:53:35.687 - [Debug] [Auth] Attacker 223.171.91.191 trying to authenticate with 'none'
2022-10-23 19:53:35.688 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 19:53:36.032 - [Debug] [EXEC] Noninteractive mode attacker command: echo > /dev/shm/ifconfig; chmod 777 /dev/shm/ifconfig; cat > /dev/shm/ifconfig
2022-10-23 19:53:36.094 - [Debug] [Connection] Attacker connected: 95.91.13.119 | Client Identification: SSH-2.0-Go
2022-10-23 19:53:36.223 - [Debug] [Auth] Attacker 223.171.91.191 trying to authenticate with 'password'
2022-10-23 19:53:36.226 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-23 19:53:36.226 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.51
2022-10-23 19:53:36.251 - [Debug] [Connection] Attacker connected: 59.1.226.211 | Client Identification: SSH-2.0-Go
2022-10-23 19:53:36.313 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2022-10-23 19:53:36.837 - [Debug] [Auth] Attacker 222.117.95.174 trying to authenticate with 'none'
2022-10-23 19:53:36.837 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 19:53:36.925 - [Debug] [Auth] Attacker 95.91.13.119 trying to authenticate with 'none'
2022-10-23 19:53:36.925 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 19:53:37.186 - [Debug] [Auth] Attacker 95.91.13.119 trying to authenticate with 'publickey'
2022-10-23 19:53:37.188 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-23 19:53:37.188 - [Debug] [Auth] Attacker: 95.91.13.119 has so far made 1 attempts. Remaining: 5 attempts
2022-10-23 19:53:37.274 - [Debug] [Auth] Attacker 222.117.95.174 trying to authenticate with 'publickey'
2022-10-23 19:53:37.275 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-23 19:53:37.275 - [Debug] [Auth] Attacker: 222.117.95.174 has so far made 1 attempts. Remaining: 5 attempts
2022-10-23 19:53:37.290 - [Debug] [Connection] Attacker connected: 58.216.8.121 | Client Identification: SSH-2.0-Go
2022-10-23 19:53:37.326 - [Debug] [EXEC] Noninteractive mode attacker command: PATH=/dev/shm/:/tmp/:./:/var/tmp/:/root/:/etc/:$PATH nc localhost 1234
2022-10-23 19:53:37.533 - [Debug] [Connection] Attacker closed the connection
2022-10-23 19:53:37.712 - [Debug] [Connection] Attacker closed the connection
2022-10-23 19:53:37.716 - [Debug] [Connection] Attacker connected: 95.91.13.119 | Client Identification: SSH-2.0-Go
2022-10-23 19:53:37.908 - [Debug] [Connection] Attacker connected: 222.117.95.174 | Client Identification: SSH-2.0-Go
